packages/app/cypress/component/support/ctSupport.ts

```diff
@@ -1,6 +1,7 @@
 import { AutIframe } from '../../../src/runner/aut-iframe'
 import { EventManager } from '../../../src/runner/event-manager'
 import type { Socket } from '@packages/socket/lib/browser'
+import '../../../src/main.scss'
```

Comment:
Why is this line here when it was not needed before?

Reply:
We needed to import tailwind and I figured it was a mistake that the rest of the shared css was not imported here.

I will remove this change and replace it by 

```js
import 'tailwindcss/tailwind.css`
```

Reply:
```suggestion
import 'tailwindcss/tailwind.css'
```

Reply:
this will probably accelerate the tests

Reply:
actually, it was a mistake...